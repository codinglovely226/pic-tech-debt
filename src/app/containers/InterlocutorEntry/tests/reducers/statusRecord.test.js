/* eslint-disable */

import statusRecordReducer from '../../reducers/communicationRecord';

describe('statusRecordReducer', () => {
  var mockedDate = new Date(2018, 3, 29, 7, 26, 26, 735);
  global.Date = jest.fn(() => mockedDate);
  it('returns the initial state', () => {
    expect(statusRecordReducer(undefined, {})).toMatchSnapshot();
  });
});
