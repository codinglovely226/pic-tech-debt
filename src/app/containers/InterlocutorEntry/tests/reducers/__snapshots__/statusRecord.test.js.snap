exports[`statusRecordReducer returns the initial state 1`] = `
Object {
  "deleted": false,
  "direction": "incoming",
  "endTimestamp": null,
  "history": Object {
    "agents": Object {},
    "features": Object {
      "descriptors": "",
      "outcome": "",
      "pastDescriptors": Object {},
      "when": "",
    },
    "investigations": Object {},
    "patient": Object {
      "age": null,
      "ageGroup": null,
      "ageUnits": null,
      "ageYears": null,
      "animalType": null,
      "category": null,
      "circumstance": null,
      "gender": null,
      "localityAddress1": null,
      "localityCity": null,
      "localityCountry": null,
      "localityGeometry": null,
      "localityISOCountry": null,
      "localityLocation": null,
      "localityName": null,
      "localityPlaceId": null,
      "localityPostcode": null,
      "localityState": null,
      "localityTelephone": null,
      "medicalRecordNumber": null,
      "medicalRecordNumberEnc": null,
      "name": null,
      "nameEnc": null,
      "occupation": null,
      "outcome": null,
      "poisonSeverityScore": null,
      "pregnantLactating": null,
      "pregnantStatus": null,
      "pregnantTrimester": null,
      "pregnantTrimesterUnknown": null,
      "riskAssessment": null,
      "topic": null,
      "weight": null,
      "weightUnits": null,
    },
    "treatments": Object {
      "pastTreatments": Object {},
      "referrals": Array [],
      "treatments": Array [],
    },
  },
  "interlocutor": Object {
    "category": "",
    "contact": "",
    "name": "",
    "nameEnc": "",
    "organisation": Object {
      "address1": "",
      "address2": "",
      "city": "",
      "country": "",
      "geometry": Object {},
      "isoCountry": "",
      "location": "",
      "name": "",
      "placeId": "",
      "postcode": "",
      "state": "",
      "telephone": "",
    },
  },
  "medium": Object {
    "detail": "",
    "detailEnc": "",
    "type": "Phone",
  },
  "noteRef": null,
  "sequenceId": 1,
  "spi": Object {
    "id": "",
    "job": "",
    "name": "",
    "org": "",
    "roles": Array [],
    "siteId": "",
  },
  "timestamp": "2018-04-29T07:26:26.735Z",
  "type": "comms",
}
`;
